Added info about v21 to the README (#3388)
OpenMage/magento-lts: README.md

UPS shut down their old CGI APIs so we removed the support for it from the Mage_Usa module.

### Between OpenMage 20.x and 21.x (unreleased, available on branch `next`)

> - PHP 8.1 is now the minimum version

Reviewer: PHP 8.1 as minimum required version

- Removed scriptaculous/dragdrop.js (#3215)
- RWD theme: updated jQuery to 3.7.0 (#3204)
- Unified CSRF configuration (#3147) and added form key validation to Contacts form (#3146)
- Removed double span element from HTML buttons (#3123)

> - Removed all deprecated Mysql4_ classes (#2730), if you have old modules that still use those classes, please use the 'shell/rename-mysql4-class-to-resource.php` to convert them

Reviewer: Removed all deprecated Mysql4_ classes (#2730)**. If you have installed old modules/extensions that use such classes then you must run in CLI /{OpenMage_ROOT}/shell/rename-mysql4-class-to-resource.php to convert them. Please backup all files before running the script**

> - Removed all deprecated Mysql4_ classes (#2730), if you have old modules that still use those classes, please use the 'shell/rename-mysql4-class-to-resource.php` to convert them

Author: ```suggestion
- Removed all deprecated Mysql4_ classes (#2730), if you have old modules/extensions that still use such classes, you have to run 'shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. backup all files before running the script.
```

Reviewer 2: That script only works on systems with "gnu grep" installed in $PATH, should probably be noted. Although.. developers probably know..

> - Removed all deprecated Mysql4_ classes (#2730), if you have old modules/extensions that still use such classes, you have to run 'shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. backup all files before running the script.

Reviewer: "backup" with capital B :)

Reviewer: I would split the paragraph in two parts 

Removed all deprecated Mysql4_ classes (#2730)**. I**f you ...

> - Removed all deprecated Mysql4_ classes (#2730), if you have old modules/extensions that still use such classes, you have to run 'shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. backup all files before running the script.

Author: ```suggestion
- Removed all deprecated Mysql4_ classes (#2730).
If you have old modules/extensions that still use such classes, you have to run 'shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. Backup all files before running the script.
```

> - Removed all deprecated Mysql4_ classes (#2730). If you have old modules/extensions that still use such classes, you have to run 'shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. Backup all files before running the script.

Reviewer: "If there are any old modules/extensions in your installation that use such classes, you must run 'shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. Backup all files before running the script"

The wording sounds better. I removed the dot at the end of the bullet in order to keep the same format used in the readme file.

OFFTOPIC: the script name it is too long compared to the other existing ones. rename-mysql4 or mysql4 were more than enough.

Reviewer: The use of the word "rename" in the name of the file is not suitable for the action that the script does. In addition, the name must be as short as possible. If we look in the /shell directory we see how simple the file names are (indexer.php, log.php). I think the most suitable would be mysql4.php.

Author: mmm no i don't agree, mysql4.php really would be impossible to understand, also, this is a one shot usage file and it's pontentially dangerous, the filename should be at least understandable

> - Removed all deprecated Mysql4_ classes (#2730). If you have old modules/extensions that still use such classes, you have to run 'shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. Backup all files before running the script.

Author: ```suggestion
- Removed all deprecated Mysql4_ classes (#2730). If there are any old modules/extensions in your installation that use such classes, you must run `shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. Backup all files before running the script
```
